src/VisualStudio.Roslyn.SDK/Roslyn.SDK/ProjectTemplates/CSharp/CodeRefactoring/Ref/CodeRefactoring.csproj

```diff
@@ -1,13 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard1.3</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="2.6.2" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="2.10" PrivateAssets="all" />
-    <PackageReference Update="NETStandard.Library" PrivateAssets="all" />
```

Comment:
What dependecies does the resulting nupkg have with this change?

Reply:
No dependencies at all. `SuppressDependenciesWhenPacking` even removes the list of declared target frameworks supported by the package.